virt/virt-2-6-release-notes.adoc

```diff
@@ -91,6 +92,28 @@ Other operating system templates shipped with {VirtProductName} are not supporte
 
 //BZ-1855182 (https://bugzilla.redhat.com/show_bug.cgi?id=1855182): [Storage] Clone could not be continued after virtctl stop the vm if the clone dv have been created for more than 3 minutes
 
+* The Containerized Data Importer (CDI) and KubeVirt depend on QEMU which does not support NFS version 3. Therefore, only NFS version 4 is supported. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1892445[*BZ#1892445*])
+
+* The name of the Fedora PVC in the `openshift-virtualization-os-images` namespace is `fedora`, instead of `fedora32`. If you populated the `fedora32` PVC in {VirtProductname} 2.5 or earlier, the virtual machine does not appear in the web console and you cannot use it to clone another virtual machine. To workaround this issue, upload a Fedora image by naming the PVC `fedora`, instead of `fedora32`. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1913352[*BZ#1913352*])
+
+* When creating a HPP boot source, the data volume is `pending` with a `WaitForFirstConsumer` status if a user creates the boot source using any method except the `Upload local file (creates PVC)` option. To resolve this issue, edit the YAML that defines the PVC in the Storage > PersistentVolumeClaims screen. Add the annotation `cdi.kubevirt.io/storage.bind.immediate.requested: "true"` to the YAML defining the persistent volume claim. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1929177[*BZ#1929177*])
```

Comment:
1. Looking through the bug I gather 'Upload local file..' is GUI. in which case needs to be marked up with asteriks rather than backticks. Might also be useful to mention it is web console option. 
2. Storage > PersistentVolumeClaims should be `*Storage* -> *Persistent Volume Claims*` However this can also be done in the CLI so do we want to be GUI specific here?
3. If answer is 'no', suggestion for last two sentences:
`+`
`** As a workaround, edit the YAML of the underlying PVC of the data volume to add the cdi.kubevirt.io/storage.bind.immediate.requested: "true" annotation:`(<-- note, literal markup dropped for annotation only because it's messing with github markup. Annotation should still be wrapped in backticks)
`+`
`[source,yaml]`
`----`
`metadata:`
`  annotations:`
`    cdi.kubevirt.io/storage.bind.immediate.requested: "true"` (<-- note: looks like github dropping the necessary yaml spacing here and for annotations: line)
`----`
To make workaround more visible and include yaml example

Reply:
Thanks Andrew, I appreciate that.